build_raspOVOS.sh

```
@@ -29,197 +98,106 @@ fi
-# Set permissions for services
-chmod 644 /home/$USER/.config/systemd/user/*.service
-chmod +x /usr/libexec/ovos-*
+# Configure user groups for audio management.
+echo "Configuring audio..."
+add_user_to_group $USER audio
+add_user_to_group $USER pipewire
+if getent group rtkit > /dev/null 2>&1; then
+    add_user_to_group $USER rtkit
+fi
+# added this per ChatGPT recommends
+chown -R $TUID:audio /run/user/$(id -u $TUID)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)